wgsl/index.bs

```diff
@@ -361,13 +356,17 @@ Issue: (dneto): Complete description of `Array<E,N>`
   <thead>
     <tr><th>Type<th>Description
   </thead>
-  <tr><td>struct<*T1*,...,*Tn*><td>An ordered tuple of *N* members of types
-                                    *T1* through *Tn*, with *N* being an
-                                    integer greater than 0.
+  <tr algorithm="structure type">
```

Comment:
Can this be pulled out, seems unrelated to pointers.

Reply:
Before this we didn't really say very strongly that structure members had names.

But yes, can do.

Reply:
I've pulled this out as #1369